.github/workflows/main.yml

```diff
@@ -39,6 +39,8 @@ jobs:
       run: |
         cd /srv/imaptest.git
         git fetch
+        git remote -v
```

Comment:
I see it's failing because of missing 'origin/main'...  maybe add `git branch -vv` here too to see what branches you actually have present.  `git remote -vv` (two v's) can be useful too, I usually use both these commands with two v's all the time, don't remember what the difference between one v and two  was though.

Reply:
Matt seems to think that we're using a local repository and not the upstream one, so we aren't able to fetch the updates which fix the failure in the notify test.  I don't know why we have our own local copy, and I don't really have the knowledge to dig in and fix it.

Reply:
I think the solution for now is - if you have access to [cyrusimap/imaptest](https://github.com/cyrusimap/imaptest), then you can update its master branch to be the upstreams latest main branch, and change this code to use master instead of main, and you're done

Reply:
Might be enough to click this "sync fork" button? (see image).  Idk if there's ramifications though.  Do we know who set up our fork?  Maybe @brong?

<img width="758" alt="Screenshot 2023-04-24 at 9 49 05 am" src="https://user-images.githubusercontent.com/10985135/233873401-97506031-8d58-44da-b90a-6e900b90f907.png">